### PR TITLE
89 stake create stake totalstaked variable in storage

### PR DESCRIPTION
**Pull request by gangov**

### CODE REVIEW

File: contracts/stake/src/tests/bond.rs [hunk 1]
```diff
@@ -164,6 +195,50 @@ fn unbond_simple() {
...
+        li.timestamp = 2000;
+    });
+    staking.bond(&user, &10_000);
+    env.ledger().with_mut(|li| {
+        li.timestamp = 4000;
+    });
+    staking.bond(&user, &10_000);
+    env.ledger().with_mut(|li| {
+        li.timestamp = 4000;
+    });
```
Comment (ueco-jb) [inline contracts/stake/src/tests/bond.rs:221]: You've set timestamp twice to the same value.

File: contracts/stake/src/tests/bond.rs
Comment (ueco-jb) [inline contracts/stake/src/tests/bond.rs:1]: Nice tests, but they don't test TotalStaked at all.

**Review by ueco-jb — CHANGES_REQUESTED** [review]
I asked you to add new query and then test that new query.

https://github.com/Phoenix-Protocol-Group/phoenix-contracts/issues/89
> Add query to read that data.
Add tests to make sure that it increases/decreases with each bond/unbond

Please pay attention to the task description...

File: contracts/stake/src/tests/bond.rs
Comment (gangov) [reply contracts/stake/src/tests/bond.rs:1]: simplified a bit

**Review by ueco-jb — APPROVED** [review]
LGTM
My only nitpick - you could rename `increase_staked` to `increase_total_staked` for better clarity (decrease as well obviously).

**Pull request merged**